build.sbt

```diff
@@ -66,6 +67,7 @@ libraryDependencies ++= Seq(
   "org.scalaz" % "scalaz-core_2.11" % "7.1.3",
   "com.github.pathikrit" %% "better-files" % "2.13.0",
   "org.liquibase" % "liquibase-core" % "3.4.2",
+  "com.squants"  %% "squants"  % "0.6.1-SNAPSHOT",
```

Comment:
Just talked about this in person but for the record I agree we should use only one and if there is already one (which I didn't realize) then there's no need to replace it.